src/main/java/hudson/plugins/release/promotion/ReleasePromotionCondition.java

```diff
@@ -34,10 +35,7 @@ public PromotionBadge isMet(AbstractBuild<?, ?> build) {
         
     }
 
-    public static void registerExtension() {
```

Comment:
Formally breaks the binary compatibility. Maybe makes sense to deprecate and to convert to NOOP-call